lib/Dialect/TTNN/Transforms/TTNNWorkarounds.cpp

```diff
@@ -150,6 +145,48 @@ static bool workaroundInputOperand(
   return true;
 }
 
+// If the data type of the output result has changed as a result of a data type
+// workaround, this method casts the data type back to the previous data type
+// by inserting a ToLayoutOp after the op output.
+//
+//                +---------+             +----------+
```

Comment:
I think the comment is self explanatory here. If you want to add more context, maybe IR would be more suited.